test_scripts/API/Additional_Submenus/additional_submenus_common.lua

```diff
@@ -0,0 +1,66 @@
+local common = require('test_scripts/Smoke/commonSmoke')
+
+local function reverseArray(arr)
+    local rev = {}
+    for i=#arr, 1, -1 do
+        rev[#rev+1] = arr[i]
+    end
+    return rev
+end
+
+function common.AdditionalSubmenu(requestParams, hmiRequestParams, parentPresent)
```

Comment:
```suggestion
function common.addSubMenu(requestParams, hmiRequestParams, parentPresent)
```

Reply:
Also, I feel that the `hmiRequestParams` field could be deduced from the `requestParams` in this function, that would make these tests a lot cleaner